plugin/client_rpc.go

```diff
@@ -187,7 +187,7 @@ func (g *hooksRPCClient) OnActivate() error {
 	_returns := &Z_OnActivateReturns{}
 
 	if err := g.client.Call("Plugin.OnActivate", _args, _returns); err != nil {
-		g.log.Error("RPC call to OnActivate plugin failed.", mlog.Err(err))
+		mlog.Error("RPC call to OnActivate plugin failed.", mlog.Err(err))
```

Comment:
Gentle reminder to check our logging [principles](https://developers.mattermost.com/contribute/server/style-guide/#log-levels) before merging this change.